x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/index.tsx

```diff
@@ -116,8 +120,11 @@ export const EPMHomePage: React.FC = () => {
   const atLeastOneUnverifiedPackageInstalled = installedPackages.some(
     (pkg) => 'savedObject' in pkg && pkg.savedObject.attributes.verification_status === 'unverified'
   );
+  const atLeastOnePackageUpgradeable = installedPackages.some(isPackageUpdatable);
 
-  const sectionsWithWarning = (atLeastOneUnverifiedPackageInstalled ? ['manage'] : []) as Section[];
+  const sectionsWithWarning = (
```

Comment:
Since we have multiple cases where a section can be flagged with a warning badge it probably makes sense to add an `EuiTooltip` to each so the user can get a little additional insight into why they're seeing a warning badge.